### PR TITLE
dx-1050

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/api-operations.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+        {
+            "method": "{{ api_method }}",
+            "href": "{{ api_href }}",
+            "rel": "{{ api_operation }}",
+            "contentType": "{{ api_contentType }}"
+        }
+
+
+{%- endcapture -%}
```
Comment (payex-dev) [inline _includes/api-operations.md:24]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5773197)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5773197&bid=18896136)

File: _includes/api-operations.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+{{code}}
```
Comment (payex-dev) [inline _includes/api-operations.md:25]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5773197)

File: _includes/api-operation.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{%- assign api_resource = include.api_resource  | default: "UNKNOWN_RESOURCE" -%}
+{%- assign operation = include.operation -%}
+{%- assign content_type = include.content_type | default: "application/javascript" -%}
```
Comment (asbjornu) [inline _includes/api-operation.md:3]:
```suggestion
{%- assign content_type = include.content_type | default: "application/json" -%}
```

File: _includes/api-operation.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+{%- capture href -%}
+{{- page.api_url }}/psp/{{ api_resource }}/payments/{{ page.payment_id }}
+{%- endcapture -%}
+{%- if href_tail != nil && href_tail != empty -%}
+    {%- assign href = href | append: '/' | append: href_tail -%}
+{%- endif -%}
+{%- assign operation_begins_with = operation | split: "-" | first -%}
+{%- case operation_begins_with -%}
+    {%- when "view" or "redirect" or "failed" or "paid" or "aborted" -%}
+        {%- assign api_method = "GET" -%}
```
Comment (asbjornu) [inline _includes/api-operation.md:14]:
```suggestion
    {%- when "view" -%}
        {%- assign api_method = "GET" -%}
        {%- assign content_type = "application/javascript" -%}
    {%- when "redirect" -%}
        {%- assign api_method = "GET" -%}
        {%- assign content_type = "text/html" -%}
    {%- when "failed" or "paid" or "aborted" -%
        {%- assign api_method = "GET" -%}
```

**Pull request merged**